site/src/components/range-anatomy.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { AnatomyWrapper, Host, Part, Slot } from './anatomy-components'
+
+const RangeAnatomy = () => {
+  return (
+    <AnatomyWrapper>
+      <Host name="input type=range">
```

Comment:
Where is the range value(s)?

Reply:
In [CSS forms this function](https://drafts.csswg.org/css-forms-1/#the-control-value-function) looks akin to what I'm referring to but we would follow what we've done with `select` and produce an element for this.